ui/ops/src/routes/ops/security-events/SecurityEventsTable.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div class="ml-2">
+    <v-row v-if="!props.overviewPage" class="mt-0 ml-0 pl-0">
+      <h3 class="text-h3 pt-2 pb-6">Security Events</h3>
+      <v-spacer/>
+    </v-row>
+
+    <content-card :class="['px-8', {'mt-8 px-4': !props.overviewPage}]">
+      <v-data-table-server
+          :headers="allHeaders"
+          v-bind="tableAttrs"
+          disable-sort
+          :items-length="queryTotalCount"
+          :row-props="rowProps"
+          class="pt-4">
+        <template #top>
+          <v-row
+              :class="[
+                'd-flex flex-row align-center mb-2 mt-1 ml-0 pl-0 mr-1'
+              ]">
+            <h3 v-if="props.overviewPage" class="text-h4">
+              Security Events
+            </h3>
+            <v-spacer/>
+          </v-row>
+        </template>
+        <template #item.createTime="{ item }">
+          {{ timestampToDate(item.securityEventTime).toLocaleString() }}
+        </template>
+      </v-data-table-server>
+    </content-card>
+  </div>
+</template>
+<script setup>
+import {timestampToDate} from '@/api/convpb';
+import ContentCard from '@/components/ContentCard.vue';
+import {useSecurityEventsCollection} from '@/composables/securityevents.js';
+import {useDataTableCollection} from '@/composables/table.js';
+import {useCohortStore} from '@/stores/cohort.js';
+import {useSidebarStore} from '@/stores/sidebar';
+import {computed, onUnmounted, ref} from 'vue';
+
+const props = defineProps({
+  overviewPage: {
+    type: Boolean,
+    default: false
+  }
+});
+
+const sidebar = useSidebarStore();
```

Comment:
I can't see that the sidebar is used for anything with this table, I can't find a way to open the sidebar.